docs/core/tools/global-json.md

```diff
@@ -33,30 +35,103 @@ Note that this field:
 - Doesn't have globbing support, that is, the full version number has to be specified.
 - Doesn't support version ranges.
 
-The following example shows the contents of a *global.json* file:
+#### allowPrerelease
+
+- Type: `boolean`
+
+- Available since: .NET Core 3.0 SDK.
+
+Indicates whether the SDK resolver should consider prerelease versions when selecting the SDK version to use.
+
+If you don't set this value explicitly, the default value depends on whether you're running from Visual Studio:
+
+- If you're **not** in Visual Studio, the default value is `true`.
+- If you are in Visual Studio, it uses the prerelease status requested. That is, if you're using a Preview version of Visual Studio or you set the **Use previews of the .NET Core SDK** option (under **Tools** > **Options** > **Environment** > **Preview Features**), the default value is `true`; otherwise, `false`.
+
+#### rollForward
+
+- Type: `string`
+
+- Available since: .NET Core 3.0 SDK.
+
+The roll-forward policy to use when selecting an SDK version, either as a fallback when a specific SDK version is missing or as a directive to use a later version. A [version](#version) must be specified with a `rollForward` value, unless you're setting it to `latestMajor`. 
+
+To understand the available policies and their behavior, consider the following definitions for an SDK version in the format `x.y.znn`:
+
+- `x` is the major version.
+- `y` is the minor version.
+- `z` is the feature band.
+- `nn` is the patch version.
+
+The following table shows the possible values for the `rollForward` key:
```

Comment:
In the table, I think available is better than higher, but I don't feel strongly about this. I think the change from lastest to highest for the "latest.." series is important.

Reply:
Just to make sure I understand since your suggestions were changing available to higher, did you mean to say that higher is better than available instead?

Reply:
Yep. Thanks for confirming.